src/site/content/en/blog/webcodecs/index.md

```diff
@@ -145,46 +151,50 @@ let config = {
   codec: 'vp8',
   width: 640,
   height: 480,
-  bitrate: 8_000_000,     // 8 Mbps
+  bitrate: 2_000_000,     // 2 Mbps
   framerate: 30,
 };
 
 let encoder = new VideoEncoder(init);
 encoder.configure(config);
 ```
 
-After the encoder has been set up,  it's ready to start accepting frames. When
-frames are coming from a media stream, the callback given to `VideoTrackReader.start()` will pump frames into
-the encoder, periodically inserting
-[keyframes](https://en.wikipedia.org/wiki/Key_frame#Video_compression) and
-checking that the encoder is not overwhelmed with incoming frames.
+After the encoder has been set up, it's ready to start accepting frames via `encode()` method.
 Both `configure()` and `encode()` return immediately without waiting for the
 actual work to complete. It allows several frames to queue for encoding at the
-same time. But it makes error reporting somewhat cumbersome. Errors are reported
-either by immediately throwing exceptions or by calling the `error()`
+same time, `encodeQueueSize()` shows how many requests are waiting in the queue waiting
+for previous encodes to finish.
+Errors are reported either by immediately throwing exceptions or by calling the `error()`
```

Comment:
> Some errors are easy to detect immediately

Possibly misleading. Ease of detection is not our criteria, layering is. Errors are thrown immediately for violations of the API contract, `error()` is called for problems encountered in the codec implementation.